src/Sentry.Bindings.Cocoa/buildTransitive/Sentry.Bindings.Cocoa.targets

```diff
@@ -0,0 +1,42 @@
+<Project>
+
+  <!--
+    Workaround to ensure the Sentry Cocoa SDK is present when using iOS "Hot Restart" deployments on Windows
+    Based on https://github.com/xamarin/xamarin-macios/pull/16605
+    The only differences are:
+      - We test that $(HotRestartPackageResourcesDir) is undefined, because if it exists that means the PR has been merged and this target is no longer required.
```

Comment:
That's a really neat thing to do.

Reply:
Yep. Otherwise we risk having our own workaround conflict with the real fix when it is released.